### PR TITLE
Custom biome source and biome layers

### PR DESCRIPTION
**Pull request by valoeghese**
I had to manually merge with the master branch :irritatered:

A big step in the progress of the dimension.

### CODE REVIEW

File: src/main/java/com/fabriccommunity/spookytime/SpookyTime.java [hunk 1]
```diff
@@ -1,20 +1,32 @@
...
 		SpookyEntities.init();
 		SpookyBlocks.init();
 		SpookyItems.init();
 		SpookyCommands.init();
-    SpookyBiomes.init();
-    SpookyDimensions.init();
+		SpookyBiomes.init();
+		SpookyWorldGen.init();
+		SpookyDimensions.init();
+		LOGGER.info("Have a spoOOoOky time!");
```
Comment (valoeghese) [inline src/main/java/com/fabriccommunity/spookytime/SpookyTime.java]: Also this

File: src/main/java/com/fabriccommunity/spookytime/common/SpookyWorldGen.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
+package com.fabriccommunity.spookytime.common;
+
+import com.fabriccommunity.spookytime.common.world.SpookyBiomeGroup;
+import com.fabriccommunity.spookytime.common.world.layer.AddSubBiomesLayer;
+
+import net.fabricmc.fabric.api.biomes.v1.OverworldBiomes;
+import net.minecraft.world.biome.Biome;
+
+public class SpookyWorldGen {
```
Comment (valoeghese) [inline src/main/java/com/fabriccommunity/spookytime/common/SpookyWorldGen.java:9]: This is where you mess with the dimension generation

File: src/main/java/com/fabriccommunity/spookytime/SpookyTime.java (hunk 1 above)
Comment (valoeghese) [reply src/main/java/com/fabriccommunity/spookytime/SpookyTime.java]: Ak9 doesn't like this line of code, and it's not related to the PR itself, so I'll remove it from the PR

**Review by ImAK9 — APPROVED** [review]
Looks good to me. Review it and I’ll merge.

**Review by Shnupbups — APPROVED** [review]

**Pull request merged**